apps/web/package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "next dev",
     "build:prod": "next build",
     "start": "next start",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -p 6007",
```

Comment:
Did this have to be changed?

Reply:
To have two storybooks running the same time -yes. Both web and main storybook